### ISSUE TITLE
ipywidgets don't seem to work on mybinder.org

### ISSUE THREAD

**Issue by sg-s** [jupyterhub/binderhub#840]
I have a python notebook [here])https://github.com/sg-s/rent-or-buy) that uses ipywidgets to enable interactivity. 

However, on binder, the notebook renders without any errors, but the widgets seem "dead", and don't update plots.

[Link to binder live notebook](https://mybinder.org/v2/gh/sg-s/rent-or-buy/master?filepath=rent-or-buy.ipynb)

* Is this expected behaviour?
* Can ipywidgets be used on binder?
* If not, what tools exist for interactivity on binder?

Thanks in advance

**Comment by betatim** [comment]
Hi!

widgets should work. If I launch https://github.com/binder-examples/requirements I can paste the following simple widget code:
```python
import ipywidgets as widgets
widgets.IntSlider()
```
and get a slider that slides.

TL;DR: It should just work.

---

As a general comment: for support questions like this (discussions that aren't about making changes to the binderhub repo itself) it is better to create a thread on http://discourse.jupyter.org/ as it has more readers and is indexed by google so future readers will have an easier time finding similar issues/solutions.

**Comment by betatim** [comment]
If I replace `%matplotlib notebook` with `%matplotlib inline` in your notebook I get a plot in your notebook. So maybe this is a question of how to configure matplotlib (figures)?

**Comment by sg-s** [comment]
HI, I should have been more clear.

The sliders *do* render in my notebook, as do the plots. But moving the sliders doesn't update the plot.

**Comment by sg-s** [comment]
OK, I fixed the problem as I was missing a `plt.show()` 

thanks for your help!
